cmd/pw_ws_broker/main.go

```diff
@@ -48,19 +47,16 @@ func main() {
 			Name:    "rabbitmq",
 			Aliases: []string{"source"},
 			Usage:   "A place to fetch data from. amqp://user:pass@host:port/vhost?ttl=60",
-			Value:   "amqp://guest:guest@rabbitmq:5672/pw",
 			EnvVars: []string{"RABBITMQ", "SOURCE"},
 		},
 		&cli.StringFlag{
 			Name:    "nats",
-			Usage:   "Nats.io URL for fetching and publishing updates.",
-			Value:   "nats://guest:guest@nats:4222/",
+			Usage:   "Nats.io URL for fetching and publishing updates. nats://guest:guest@host:4222/",
```

Comment:
The defaults were just getting in the way.

My problem was that i could not specify via flags a working URL for rabbit/nats/redis

moved them to a comment, there is no need now to do a fancy "is flag set" dance